app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,11 @@
   </head>
 
   <body>
+    <% flash.each do |type, msg| %>
+      <div id="flash_message">
+        <%= msg %>
+      </div>
+    <% end %>
```

Comment:
We're going to have to work on these flash messages because currently there will be duplicate messages